README.rst

```diff
@@ -65,12 +65,15 @@ MO-tilretninger af datamodellen i LoRa
 
 For at få datamodellen i LoRa til at afspejle datamodellen i MO skal
 LoRAs konfiguration justeres så den anvender en anden
-databaseopsætning::
```

Comment:
We should probably explicitly mention that the file lives in the `oio_rest` repository, for when changes need to be made to it.